### DISCUSSION TITLE
Inline download icon

### DISCUSSION THREAD

**Discussion by ashut0shk** [Docile-Alligator/Infinity-For-Reddit#937] [Ideas]
Hi

Please add inline download feature for media in the main feed itself. 

- Currently to download, one has to click/open on the image/video and then click the download icon. 

- Expected, a download icon right with the feed itself, probably near upvote or downvote icons, to download that post media.

**Comment by Docile-Alligator** [comment]
This is a good idea. But the icons may overlap each other.

**Comment by ashut0shk** [comment]
@Docile-Alligator With the current layout, being all the controls on the center of the media file, I think, the bottom area could be utilized for download icon without being interfering.
